PoshSSDTBuildDeploy/Functions/InstallMicrosoftDataToolsMsBuild.ps1

```diff
@@ -1,16 +1,15 @@
 
 
 function Install-MicrosoftDataToolsMSBuild {
-
-    param ( [string] $WorkingFolder
-        , [string] $DataToolsMsBuildPackageVersion
+    [cmdletbinding()]
+    param ( 
+        [parameter(Mandatory)]
```

Comment:
Added Mandatory property so that you can have cleaner code and remove the if statement on lines 9-11